src/plugins/feature_catalogue/public/plugin.test.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { registryMock } from './plugin.test.mocks';
+import { FeatureCataloguePlugin } from './plugin';
+
+describe('FeatureCataloguePlugin', () => {
+  beforeEach(() => {
+    registryMock.setup.mockClear();
+    registryMock.start.mockClear();
+  });
+
+  describe('setup', () => {
+    test('wires up and returns registry', async () => {
```

Comment:
optional: not sure those are really useful tests if TS enforces the contract

Reply:
I can go either way here, but I prefer some coverage here over nothing.